.travis.yml

```diff
@@ -10,7 +10,7 @@ install:
   - pip install -r requirements.txt
 script:
   - ./tests/scripts/unit_tests.sh
-  - ./manage.py lambda test --processor all
+  - ./manage.py test rules
```

Comment:
I think we need test for classifier in CI as well.

Reply:
the `rules` test implicitly tests the classifier as well. the stages are:

- `manage.py test classifier` --> tests the classifier only
- `manage.py test rules` --> tests the classifier AND rules engine
- `manage.py test live` --> tests the classifier, rules engine, AND tries to send alerts

Reply:
Aha, thanks!